test/gtest/uct/ib/test_sockaddr.cc

```diff
@@ -398,12 +502,34 @@ UCS_TEST_P(test_uct_cm_sockaddr, cm_query)
     }
 }
 
-UCS_TEST_P(test_uct_cm_sockaddr, listen_and_init_connect)
+UCS_TEST_P(test_uct_cm_sockaddr, cm_open_listen_close)
+{
+    UCS_TEST_MESSAGE << "Testing " << m_listen_addr
+                     << " Interface: " << GetParam()->dev_name;
+
+    cm_listen_and_connect();
+
+    wait_for_bits(&m_cm_state, TEST_CM_STATE_SERVER_CONNECTED |
+                               TEST_CM_STATE_CLIENT_CONNECTED);
+    EXPECT_TRUE(ucs_test_all_flags(m_cm_state, (TEST_CM_STATE_SERVER_CONNECTED |
+                                                TEST_CM_STATE_CLIENT_CONNECTED)));
+
+    cm_disconnect(m_client);
```

Comment:
shouldn't we check `TEST_CM_STATE_*_DISCONNECTED` states here?

Reply:
you mean instead of checking them inside cm_disconnect()?

Reply:
Missed this, inside cm_disconnect() is ok.